### PR TITLE
Fix broken google driver

### PR DESCRIPTION
**Pull request by dgageot**
Signed-off-by: David Gageot <david@gageot.net>

### CODE REVIEW

**Comment by dgageot** [comment]
@dmp42 How is it possible to decrease coverage by adding tests?...

**Comment by dmp42** [comment]
@dgageot because `go list` doesn't get to compute_utils  it seems.
I'll have a look at what's going on.

**Comment by dgageot** [comment]
@dmp42 @nathanleclaire except for the coverage bug I think it's ready to merge. Wdyt?

**Comment by dmp42** [comment]
@dgageot explanation: code with no test files will not appear in the coverage report.

Thus *adding* a test file providing coverage below the average for a file that wasn't in previously will bring the overall coverage down.

This is silly - and I would blame go cover for that (or maybe coverall).

Let me know if you want to get a crack at that but it should be low priority.
A simple workaround is to add empty test files for all go files - but we have been there and it's messy.
I would rather have proper tests :-)

Bottom-line: LGTM

@nathanleclaire wdyt?

**Comment by nathanleclaire** [comment]
I don't mind the effect on coverage, but I do have a few nitpicks about code style.  I'll write inline

File: drivers/google/compute_util.go [hunk 1]
```diff
@@ -277,6 +275,17 @@ func (c *ComputeUtil) createInstance(d *Driver) error {
 	return nil
 }
 
+// tags computes the tags for the instance.
+func tags(d *Driver) []string {
```
Comment (nathanleclaire) [inline drivers/google/compute_util.go]: Why not make this a method directly on the driver and name it something slightly more descriptive (i.e. `func (d *Driver) parseTags() []string` ?

Comment (dgageot) [inline drivers/google/compute_util.go]: @nathanleclaire ok for the better name. Making the method a function on the driver gives too much importance to it I think. I'd rather leave it as is.

**Comment by dmp42** [comment]
Nitpicks are fixed. LGTM.

**Comment by nathanleclaire** [comment]
Hm, I meant for it to be `d.parseTags` instead of `parseTags(d)`, but maybe that's splitting hairs.  It's OK.

**Pull request merged**